navigation/create_control/config/create_2_control.yaml

```
@@ -0,0 +1,48 @@
+# http://wiki.ros.org/diff_drive_controller for more parameters.
+
+# gazebo_ros_control_params.yaml
+gazebo_ros_control/pid_gains:
+  wheel_left_joint: {p: 1.0, i: 0, d: 0}
+  wheel_right_joint: {p: 1.0, i: 0, d: 0}
+
+mobile_base_controller:
+  type        : "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'wheel_left_joint'
+  right_wheel : 'wheel_right_joint'
+  publish_rate: 30.0               # default: 50
+  pose_covariance_diagonal : [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+  enable_odom_tf: true
+  # Wheel separation and diameter. These are both optional.
+  # diff_drive_controller will attempt to read either one or both from the
+  # URDF if not specified as a parameter
+  wheel_separation : 0.235
+  wheel_radius : 0.036
+
+  # Wheel separation and radius multipliers
+  wheel_separation_multiplier: 1.0 # default: 1.0
+  wheel_radius_multiplier    : 1.0 # default: 1.0
+
+  # Velocity commands timeout [s], default 0.5
+  cmd_vel_timeout: 0.2
+
+  # Velocity and acceleration limits
+  # Whenever a min_* is unspecified, default to -max_*
+  linear:
+    x:
+      has_velocity_limits    : true
+      max_velocity           : 0.3  # m/s
+      min_velocity           : -0.2 # m/s
```

Aren't these too slow?

```suggestion
      max_velocity           : 0.5  # m/s
      min_velocity           : -0.4  # m/s
```

---

Same for other limits.

---

Done